populate_dev_data_handler.go

```diff
@@ -68,6 +73,41 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
         },
     }
 
+    // Get the redirects, but don't follow them. Mimics http.ErrUseLastResponse (golang v1.7)
+    client := urlfetch.Client(ctx)
+    client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+        return errUseLastResponse
+    }
+
+    for _, browserName := range([]string {
+        "chrome",
+        "edge",
+        "firefox",
+        "safari",
+    }) {
+        jsonURL := "https://wpt.fyi/json?platform=" + browserName
```

Comment:
We seem to use the base URL `https://wpt.fyi` quite a bit now in our code, can we store it as a constant in `main.go`?

Reply:
TODO added.